src/components/divider/divider.css

```
@@ -15,4 +15,5 @@
   height: 100%;
   border-left: solid var(--width) var(--color);
```

```suggestion
  border-inline-start: solid var(--width) var(--color);
```

_Very_ minor, but I figure if we're setting `min-block-size`, we may want to use the logical size here, too.